### DISCUSSION TITLE
Laravel style loading error with mode jit

### DISCUSSION THREAD

**Discussion by 3Gdesigner** [tailwindlabs/tailwindcss#4275] [Help]
### What version of Tailwind CSS are you using?

2.1.1

### What version of Node.js are you using?

v14.16.1

### What browser are you using?

microsoft Edge

### What operating system are you using?

windows 10

### Reproduction repository

public

### Describe your issue

Hello guys from Tailwindlabs, I don't know if this is the right channel, but I'm having errors of not compiling the CSS style in my laravel project using the mode: built-in 'git' looks reminding me that I'm new to laravel

**Comment by adamwathan** [comment]
Hey @3Gdesigner! Mind sharing a GitHub repo that shows what you've got so far? Much easier to help if we can see the code 👍🏻
